app/scripts/proactive/templates/job-template.html

```diff
@@ -21,7 +21,7 @@
     <% if (job["Generic Info"] && job["Generic Info"].length>0) { %>
     <genericInformation>
     <% _.each(job["Generic Info"], function(gi) { %>
-            <info name="<%=gi["Property Name"]%>" value="<%=gi["Property Value"]%>"/>
+            <info name="<%=gi["Property Name"]%>" value="<%= _.escape(gi["Property Value"]) %> "/>

```

Comment:
I tested Scheduler submission as well. It fails for the same reason:

Invalid workflow
At [tag:job] : Element type "info" must be followed by either attribute specifications, ">" or "/>".